app/models/book.rb

```diff
@@ -0,0 +1,34 @@
+class Book < OpenStax::Cnx::V1::Book
+
+  RAP_URL_WITHOUT_PIPELINE = "https://openstax.org/apps/archive"
+  LEGACY_ARCHIVE_URL = "https://openstax.org/contents"
```

Comment:
These were hardcoded before as `https://archive.cnx.org`, not sure if we want them to be configurable?